include/triton/Dialect/TritonGPU/IR/TritonGPUOps.td

```diff
@@ -35,7 +35,11 @@ def TTG_AsyncWaitOp : TTG_Op<"async_wait"> {
 
   let extraClassDeclaration = [{
     static bool isSupported(int computeCapability) {
-      return computeCapability >= 80;
+      // ifdef USE_ROCM
```

Comment:
Maybe restore the USE_ROCM ifdef here?

Reply:
I will try to find other way

Reply:
actually, i manually ported these changes into a fresh checkout and the ifdefs worked for me.

Reply:
Using #ifdef to source parts is not common in .td files so I would like to avoid it. Added #ifdef in the only place `isSupported` used. Actually compilation asserts in case `AsyncWaitOp` is generated so we'll catch it in the case.

Reply:
Makes sense.  Approved.  Thanks for fixing this the right way.